tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "strict": true,
     "skipLibCheck": true,
     "downlevelIteration": true,
-    "target": "es2020",
+    "target": "esnext",
```

Comment:
Any reason why we need to update this?

Reply:
needed for one of the newer language supported features, but I confess I lost track of each one. do you see an issue bumping to next?